src/index.js

```diff
@@ -1,42 +1,39 @@
 import _ from "lodash";
 import "./style.css";
 
-const taskArray = [
-  {
-    index: 0,
-    descriptions: "I  will code",
-    completed: true,
-  },
-  {
-    index: 1,
-    descriptions: "i love code",
-    completed: false,
-  },
-  {
-    index: 2,
-    descriptions: "i like webpack",
-    completed: true,
-  },
-  {
-    index: 3,
-    description: "I love microverse",
-    completed: false,
-  },
-];
+import addedTask from "./module/class.js";
```

Comment:
- [ ] `required`
- [ ] Good job so far 👏 
- [ ] However your project is not running as it should because of the module
- [ ] please see the screenshot below 🥂 

![error](https://user-images.githubusercontent.com/66230306/195430113-0fea5158-03c8-4643-a995-2be3af71dab8.png)